### PR TITLE
Add petition page

### PR DESCRIPTION
**Pull request by EricBoersma**

### CODE REVIEW

File: routes/campaigners.rb [hunk 1]
```diff
@@ -20,6 +29,54 @@ class Campaigners < Cuba
       res.redirect "/"
     end
 
+    on "petitions/new" do
+      render("petition/new",
+             title: "Actionite | Create New Petition",
```
Comment (ceciliarivero) [inline routes/campaigners.rb]: To not have to repeat "Actionite | " in the `title:` parameter, we can write the <title> like this in the master templates:
`<title>Actionite | {{ title }}</title>`

File: routes/campaigners.rb [hunk 2]
```diff
@@ -20,6 +29,54 @@ class Campaigners < Cuba
...
+        petition.body = params['body']
+        petition.mobile_body = params['mobile_body']
+        petition.facebook_title = params['facebook_title']
+        petition.language = params['language']
+        petition.alt_body = params['alt_body']
+        petition.image_url = params['image_url']
+        petition.image_text = params['image_text']
+        petition.suggested_tweets = params['suggested_tweets']
+        petition.goal = params['goal']
+        petition.auto_increment_goal = params['goal']
```
Comment (ceciliarivero) [inline routes/campaigners.rb]: Shouldn't it be?
`petition.auto_increment_goal = params['auto_increment_goal']`

File: views/petition/new.mote [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<h2>Create New Petition</h2>
+<br/>
+<div class="col-lg-8">
+    {{ form }}
+</div>
+
```
Comment (ceciliarivero) [inline views/petition/new.mote]: No more comments from me :-)

**Comment by EricBoersma** [comment]
All comments addressed!

**Pull request merged**